cabal.project

```diff
@@ -1,36 +1,3 @@
-source-repository-package
-  type: git
-  location: https://github.com/con-kitty/concat.git
-  tag: 949ced997a600168205e4ad9d2f1425ccd3dc622
-  subdir: classes
-
-source-repository-package
-  type: git
-  location: https://github.com/con-kitty/concat.git
-  tag: 949ced997a600168205e4ad9d2f1425ccd3dc622
-  subdir: examples
-
--- dependency of concat-classes, concat-examples
-source-repository-package
-  type: git
-  location: https://github.com/con-kitty/concat.git
-  tag: 949ced997a600168205e4ad9d2f1425ccd3dc622
-  subdir: inline
-
--- dependency of concat-classes, concat-examples
-source-repository-package
-  type: git
-  location: https://github.com/con-kitty/concat.git
-  tag: 949ced997a600168205e4ad9d2f1425ccd3dc622
-  subdir: known
-
--- dependency of concat-classes
-source-repository-package
-  type: git
-  location: https://github.com/con-kitty/concat.git
-  tag: 949ced997a600168205e4ad9d2f1425ccd3dc622
-  subdir: satisfy
-
```

Comment:
I would really like to restore these for now. But ... I think restoring these doesn’t let us get rid of `concat` in the flake, right? So we’re back to some duplication.

Another feature request for `cabalProject2Nix`!

Reply:
Okay with that duplication (hope that it will be temporary as official concat accepts our modification eventually) :-)
I restored that source-repository-package. Now inside `nix develop .#` shell, `concat` is pre-installed with our version, but `cabal v2-build ..` will rebuild concat again from the source. If one doesn't like that, one should temporarily comment out the source-repository-package lines in cabal.project.

Reply:
Well, the duplication is less because of the lack of concat overlay and more to allow people to dev without needing Nix or direnv.